style.css

```diff
@@ -42,6 +42,45 @@ header {
   margin-bottom: 0;
 }
 
+.mobile-lists {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  padding-top: 30px;
+  width: 100%;
+  height: 100%;
+  color: white;
+  background-color: rgba(85, 101, 243, 0.7);
+  background-color: rgba(41, 58, 208, 0.8);
+  backdrop-filter: blur(5px);
+  transition: all 0.3s linear;
+}
+
+.mobile-list {
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
+  margin: auto;
+  padding: 20px;
+  width: 80%;
+  font-size: 2rem;
+}
```

Comment:
Even though the mobile menu functionality is working as expected but you need to change the following:

- [ ] Kindly make the mobile menu background take the whole width of the page, so kindly remove the white margin at the top.
- [ ] Please add the correct font family and the correct placement for the mobile menu options by moving them to the left to match the given design.


| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/231472729-e512672b-1ac2-4a7d-af66-d58d97d3a177.png)  | ![image](https://user-images.githubusercontent.com/68796232/231472689-35b8c00e-e9c6-4cee-a9ea-922776c625ad.png)  |